core/templates/pages/classroom-admin-page/classroom-admin-page.component.ts

```diff
@@ -252,6 +329,210 @@ export class ClassroomAdminPageComponent implements OnInit {
   ngOnInit(): void {
     this.getAllClassroomIdToClassroomName();
   }
+
+  getTopicDependencyByTopicName(
```

Comment:
The function seems to imply it is a getter, i.e. it returns something but really it is just setting a few things. Should we change it to  `setTopicDependencyByTopicName` instead?

Reply:
Updated